app/src/main/java/org/stepic/droid/di/step/StepComponent.kt

```diff
@@ -16,8 +14,4 @@ interface StepComponent {
     }
 
     fun commentsComponentBuilder(): CommentsComponent.Builder
```

Comment:
StepComponent is not needed anymore

Reply:
This is still needed in old comments implementation